Kernel/Time/TimeManagement.cpp

```diff
@@ -231,7 +231,10 @@ Time TimeManagement::boot_time()
 #if ARCH(X86_64)
     return RTC::boot_time();
 #elif ARCH(AARCH64)
-    TODO_AARCH64();
+    static auto number_of_times_message_has_been_shown = 0;
+    if (++number_of_times_message_has_been_shown < 5)
+        dbgln("TimeManagement: FIXME: Return correct boot time");
+    return now();
```

Comment:
This should be `0`, not `now()`, IMO.
Code might (And should be allowed to) assume that this always returns the same value during the same boot.

Reply:
Yeah, I wasn't really sure what was better. Changed it to return `0` and instead of printing a FIXME to the console, I added a FIXME in the code instead.